src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends ColorSupplier implements Figure {

```

Comment:
The class declaration begins with an empty line. Please remove the empty line at the start of the class definition.